src/Westwind.Globalization/DbResourceDataManager/DbResourceDataManagers/DbResourceMySqlDataManager.cs

```diff
@@ -196,7 +213,7 @@ PRIMARY KEY (`pk`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
 
 INSERT INTO `{0}` (ResourceId,Value,LocaleId,ResourceSet) VALUES ('HelloWorld','Hello Cruel World (MySql)','','Resources');
-INSERT INTO `{0}` (ResourceId,Value,LocaleId,ResourceSet) VALUES ('HelloWorld','Hallo schn�de Welt (MySql)','de','Resources');
+INSERT INTO `{0}` (ResourceId,Value,LocaleId,ResourceSet) VALUES ('HelloWorld','Hallo schnöde Welt (MySql)','de','Resources');
```

Comment:
Although it appears correct in GitHub, locally the ö is mojibake in this file. This line and line 227 are changed to fix the mojibake.